pyteal/ast/subroutine.py

```diff
@@ -21,15 +34,13 @@ def __init__(
         self,
         implementation: Callable[..., Expr],
         returnType: TealType,
-        nameStr: str = None,
+        nameStr: Optional[str] = None,
+        routerRegistrable: bool = False,
```

Comment:
```suggestion
        router_registrable: bool = False,
```
I feel guilty for even suggesting this...

Reply:
oh boy, I should go back to pep8 and check the formatting...

Reply:
👍